### ISSUE TITLE
galera play fails to stop mariadb on initial node

### ISSUE THREAD

**Issue by mancdaz** [rcbops/ansible-lxc-rpc#222]
```
TASK: [galera_bootstrap | Stop mariadb] ***************************************
failed: [355163-node7_galera_container-40db4417] => {"failed": true}
msg:  * Stopping MariaDB database server mysqld
   ...fail!
```

Possibly related to https://github.com/rcbops/ansible-lxc-rpc/pull/214, but probably not

**Comment by mancdaz** [comment]
Seems like the `policy-rc.d` file that we are dropping, is not preventing mysql from being started upon package install

**Comment by mancdaz** [comment]
Actually, it seems now that we are installing the mysql packages before we drop in the `policy-rc.d file``. This is done here:

```
TASK: [container_common | Ensure container packages are installed] ************
ok: [355164-node8_galera_container-ac2d5be8] => (item=mariadb-client,mariadb-galera-server-5.5,galera,python-software-properties,software-properties-common,debconf-utils,rsync,xtrabackup)
```

**Comment by andymcc** [comment]
Found it, and fixed it will post patch now - basically we moved the install of mariadb/galera out to common which fails because for galera we do a couple things (like preseed and ensure it doesnt start on install before installing it - but when we get to that its already installed)

**Comment by mancdaz** [comment]
closed via #224
